server.js

```diff
@@ -1,30 +1,69 @@
 import express from "express";
 import cors from "cors";
+import eventsData from "./data/events.json";
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
 const port = process.env.PORT || 8080;
 const app = express();
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+app.get("/events", (req, res) => {
+  let filteredEvents = eventsData;
+
+  const { city, category } = req.query;
+
+  if (city) {
+    filteredEvents = filteredEvents.filter(event => event.city.toLowerCase().includes(city.toLowerCase()));
+  }
+
+  if (category) {
+    filteredEvents = filteredEvents.filter(event =>
+      event.category.some(cat => cat.toLowerCase().includes(category.toLowerCase()))
+    );
+  }
+
+  res.json(filteredEvents);
+});
```

Comment:
Nice and RESTful making use of query params to be able to chain the filters like this ⭐